flight/intake_gps.py

```diff
@@ -0,0 +1,83 @@
+"""
+Contains the extract_gps() function for extracting data out of
+a provided target waypoint data JSON file for the Argonia Cup competition.
+"""
+from typing import Any, NamedTuple, Tuple
+import json
+
+
+class Waypoint(NamedTuple):
+    """
+    NamedTuple storing the data for a single waypoint.
+
+    Attributes
+    ----------
+    latitude : float
+        The latitude of the waypoint.
+    longitude : float
+        The longitude of the waypoint.
+    altitude : float
+        The altitude of the waypoint.
+    """
+
+    latitude: float
+    longitude: float
+    altitude: float
+
+
+def extract_gps(path: str) -> Tuple[Waypoint, float]:
```

Comment:
Guess I missed this the first time, but this is the old way of typing tuples. In Python 3.10+, you can do `tuple[Waypoint, float]` and avoid doing the import from typing.